app/src/main/java/org/wikipedia/talk/UserTalkPopupHelper.kt

```diff
@@ -73,6 +117,9 @@ object UserTalkPopupHelper {
                     R.id.menu_user_contributions_page -> {
                         activity.startActivity(UserContribListActivity.newIntent(activity, title.text))
                     }
+                    R.id.menu_user_thank -> {
+                        pageId?.let { showThankDialog(activity, title, revisionId!!, pageId) }
```

Comment:
Would it be possible to use an `if` statement to check both `revisionId` and `pageId` to see if any of them is `null`?